### DISCUSSION TITLE
SMTCOMP 2023

### DISCUSSION THREAD

**Discussion by TrevorHansen** [stp/stp#466] [Announcements]
We had a good showing at SMTCOMP 2023, winning all the QF_BV single-query divisions:
`https://smt-comp.github.io/2023/results/qf-bitvec-single-query`

It's a result of lots of hard work over the years. Thanks to everyone who has contributed!

**Comment by msoos** [comment]
Wow, that's REALLY good performance! I now wanna understand STP a bit more... I have been working on a higher-level system ( https://github.com/ethereum/hevm/ ) using an SMT solver (currently, CVC5 & Z3), and it's kinda fun operating at a higher level than SAT :) Well done on the results, great stuff!!

**Comment by hellovijay** [comment]
Amazing performance, Trevor and team!
